pkgs/development/interpreters/python/mk-python-derivation.nix

```diff
@@ -96,7 +96,10 @@
 
 , meta ? {}
 
-, passthru ? {}
+# Whether to run tests in a separate passthru.tests derivation.
+# A test derivation will be added to `passthru.tests` when `separateChecks = true`.
+, separateChecks ? false
```

Comment:
I think separateCheckPhase would be a better name. Also the s would indicate that this checks multiple things, but I think we are only run checkPhase which is usally pytest which is one check.